src/Components/Patient/models.tsx

```diff
@@ -1,188 +1,205 @@
 export interface FlowModel {
-    id?: number;
-    status?: string;
-    created_date?: string;
-    modified_date?: string;
-    deleted?: boolean;
-    notes?: string;
-    patient_sample?: number;
-    created_by?: number;
+  id?: number;
+  status?: string;
+  created_date?: string;
+  modified_date?: string;
+  deleted?: boolean;
+  notes?: string;
+  patient_sample?: number;
+  created_by?: number;
 }
 
 export interface DischargeSummaryModel {
-    email?: string;
+  email?: string;
 }
 
 export interface PatientModel {
-    id?: number;
+  id?: number;
+  name?: string;
+  age?: number;
+  allow_transfer?: boolean;
```

Comment:
Codacy found an issue: [Identifier 'allow_transfer' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5828652)